### ISSUE TITLE
New ACT Implementation Request: Webmate Accessibility Workbench

### ISSUE THREAD

**Issue by milyas-testfabrik** [w3c/wcag-act-rules#311]
This issue is created to submit a new ACT Implementation:

Tool Name: Webmate Accessibility Workbench
Webpage: https://barrierefr.ai/
By: Testfabrik Consulting + Solutions AG

Link to the EARL Report : https://webmedia.testfabrik.com/act_implementation_report/webmate_accessibility_workbench_2024_11_26.json

**Comment by daniel-montalvo** [comment]
Dear @milyas-testfabrik 

Sorry for getting to this so late.

#332 contains a preview of how your tool would be shown on the WAI website.

- [Table with all implementations](https://deploy-preview-332--wai-wcag-act-rules.netlify.app/standards-guidelines/act/implementations/) which contains several tables. You are currently listed under the "Automated tools" section.
- Specific page containing your [detailed report](https://deploy-preview-332--wai-wcag-act-rules.netlify.app/standards-guidelines/act/implementations/webmate-accessibility-workbench/).

Question: Your [report URL](https://webmedia.testfabrik.com/act_implementation_report/webmate_accessibility_workbench_2024_11_26.json) is dated. We would need a permanent URL so that the implementations can get updated automatically as you update your tool.

Is this still the URL you want us to use or there is a permanent one that you can share with us?

Best.

**Comment by milyas-testfabrik** [comment]
Hi @daniel-montalvo ,

Thank you for getting back to me and sharing the preview. Below is the permanent URL for the detailed (slightly updated) report:

Permanent Report URL: https://webmedia.testfabrik.com/act_implementation_report/webmate_accessibility_workbench.json 

I also noticed a few omissions in the `_data/wcag-act-rules/act-implementations.yml` file (which holds the metadata for ACT-listed tools). The **homepage** and **vendorHomepage** URLs are blank, and “AAA” is missing from the **standards** list. Could you please update `act-implementations.yml `with the following entries?

```
homepage: https://barrierefr.ai/
vendorHomepage: https://testfabrik.com/
standards:
  - WCAG 2.2 Level A, AA, AAA
```

One more question: once this preview is deployed, will the ACT implementation automatically be linked to our tool (“Webmate Accessibility Workbench”) on the W3C Tools List (https://www.w3.org/WAI/test-evaluate/tools/list/), or is there something we need to do on our end to establish that link?

Thanks for your help—please let me know if you need anything else. I am eagerly looking forward to seeing our tool listed on the ACT webpage.

Best regards,

**Comment by daniel-montalvo** [comment]
Hi @milyas-testfabrik 

We have updated the PR with your new report and your suggested metadata.

> One more question: once this preview is deployed, will the ACT implementation automatically be linked to our tool (“Webmate Accessibility Workbench”) on the W3C Tools List (https://www.w3.org/WAI/test-evaluate/tools/list/), or is there something we need to do on our end to establish that link?

The link to your ACT rules implementations will be added to the list page by our colleague @remibetin 

Let me know if you are happy with how the Webmate implementations show up on the above preview and then we will merge it so that it appears on the WAI website.

**Comment by milyas-testfabrik** [comment]
Dear @daniel-montalvo ,

Thank you for updating the PR with the requested changes. We are happy with how the Webmate implementations appear in the preview. Please go ahead and merge it so it can be published on the WAI website.

Thanks again for your assistance!

Best regards,

**Comment by daniel-montalvo** [comment]
Hi @milyas-testfabrik 

We finally merged the Pr and the results should be available at https://www.w3.org/WAI/standards-guidelines/act/implementations/webmate-accessibility-workbench/

I would encourage you to update your earl report periodically so that your results don't become outdated.
